app/bundles/DynamicContentBundle/Tests/Controller/DynamicContentControllerFunctionalTest.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
```

Comment:
New files should declare strict types.
```suggestion

declare(strict_types=1);

```

Reply:
OK :)